x-pack/plugins/security_solution/scripts/run_cypress/utils.ts

```diff
@@ -156,3 +159,22 @@ export const getOnBeforeHook = (module: unknown, beforeSpecFilePath: string): Fu
 
   return module.onBeforeHook;
 };
+
+/**
+ * Sets the default log level for `ToolingLog` instances created by `createToolingLogger()`:
+ * `x-pack/plugins/security_solution/common/endpoint/data_loaders/utils.ts:148`.
+ * It will first check the NodeJs `process.env` to see if an Environment Variable was set
+ * and then, if provided, it will use the value defined in the Cypress Config. file.
+ */
+export const setDefaultToolingLoggingLevel = (cypressConfigFile?: any) => {
```

Comment:
There is no point using `any` type here. `cypressConfigFile` in upstream `parallel` and `parallel_serverless` files already has type `any`. We could provide a more specific type for  `cypressConfigFile` here like

```ts
interface CypressConfigFile {
  env?: Record<string, unknown>;
}
```